.eslintrc.json

```diff
@@ -21,6 +21,16 @@
     "key-spacing": [ "error", { "beforeColon": false, "afterColon": true } ],
     "keyword-spacing": "error",
     "linebreak-style": [ "error", "unix" ],
+    "max-len": [ "warn", {
+      "code": 100,
+      "tabWidth": 4,
+      "ignorePattern": "^/// .+|// (?!eslint-.+max-len)eslint-.+|QUnit.",
```

Comment:
We should ignore all lines starting with `// eslint-`, not just the ones disable something other than `max-len`.

Reply:
This is intended for lines with an inline // eslint… (which we mostly discourage).

Reply:
Yeah, but it's seems it excludes `// eslint-disable-next-line max-len`, which seems odd. The line disabling eslint for `max-line` (among other reasons) is just as likely to itself exceed the limit as any other disable line. I think this may be trying to fix a bug that might not exist.

Reply:
Maybe drop the ignorePattern for now.